eng/scripts/live-test-resource-cleanup.ps1

```diff
@@ -102,15 +119,31 @@ function HasValidOwnerTag() {
     Write-Warning " Resource group '$($ResourceGroup.ResourceGroupName)' has invalid owner tags: $($invalidOwners -join ',')"
   }
   if ($hasValidOwner) {
-      Write-Host " Skipping tagged exception resource group '$($ResourceGroup.ResourceGroupName)' with owners '$owners'"
-      return $true
+    Write-Host " Skipping tagged resource group '$($ResourceGroup.ResourceGroupName)' with owners '$owners'"
+    return $true
   }
   return $false
 }
 
+function HasValidAliasInName([object]$ResourceGroup) {
+    # check compliance (formatting first, then validate alias) and skip if compliant
+    if ($ResourceGroup.ResourceGroupName `
+      -match "^(rg-)?((t-|a-|v-)?[a-z,A-Z]{3,15})([-_]{1}.*)?$" `
+      -and (IsValidAlias -Alias $matches[2]))
+    {
+      Write-Host " Skipping resource group '$($resourceGroup.ResourceGroupName)' starting with valid alias '$($matches[2])'"
```

Comment:
Nit: I recommend using named matches as it's much easier to maintain - especially in the face of changes. You should also use single quotes for regex, thus your regex would be:

```powershell
'^(rg-)?(?<alias>(t-|a-|v-)?[a-z,A-Z]{3,15})([-_]{1}.*)?$'
```

And your usage would look like:

```powershell
Write-Host " ... '$($Matches['alias'])`"
```